src/com/theironyard/novauc/CustomMath.java

```diff
@@ -0,0 +1,22 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by souporman on 2/7/17.
+ */
+public class CustomMath {
+    static double AreaOfCircle(double radius){
+        return (Math.PI*(radius*radius));
+    }
```

Comment:
Nice job Alvaro.  The basic requirements were met so well I have no critical feedback for you.  Awesome job with hardmode.  You could also use Math.pow() instead of (radius * radius).  But works just the same.